.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: php
 
 php:
-  - "5.3"
-  - "5.4"
-  - "5.5"
-  - "5.6"
-  - "7.0"
-  - "hhvm"
+  - "7.1"
+  - "7.2"
 
 matrix:
   allow_failures:
-    - php: "hhvm"
+    - php: "7.2"
```

Comment:
Just because of `each()` in https://github.com/ChessCom/Chess-Game/blob/develop/src/Chess/Game/ChessGame.php#L2293?

Reply:
This library is abandoned. I'm not planning on supporting 7.2 in this PR, just to make minor changes to make tests work again in my environment.